src/features/views/components/ViewDataTable/columnTypes/SurveyOptionColumnType.tsx

```diff
@@ -48,7 +48,7 @@ export default class SurveyOptionColumnType
 const Cell: FC<{ cell: SurveyOptionViewCell }> = ({ cell }) => {
   const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null);
   const { openPane } = usePanes();
-  const { orgId } = useRouter().query;
+  const { orgId, surveyId } = useRouter().query;
```

Comment:
In this case, the survey ID cannot come from the URL. It's also not possible to retrieve it from the column config, apparently. Will investigate and add more info in subsequent comments.

Try clicking "Show full submission" after hovering any of the values for Angela Davis in the rightmost columns here to see the problems: https://app-zetkin-fqxi0nifd-zetkin.vercel.app/organize/1/people/lists/126